app/src/main/java/com/ggetters/app/data/repository/notification/OfflineNotificationRepository.kt

```
@@ -20,17 +24,53 @@ class OfflineNotificationRepository @Inject constructor(
+        try {
+            dao.insert(entity)
+            Clogger.d("OfflineNotificationRepo", "✅ Notification inserted to DAO")
+            
+            // Verify it was actually inserted
+            val inserted = dao.getById(entity.id)
+            if (inserted != null) {
+                Clogger.d("OfflineNotificationRepo", "✅ VERIFIED: Found in DB (userId: ${inserted.userId}, teamId: ${inserted.teamId})")
+            } else {
+                Clogger.e("OfflineNotificationRepo", "❌ ERROR: NOT FOUND in DB immediately after insert!")
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)